debian/rules

```diff
@@ -15,7 +15,7 @@ override_dh_auto_test:
 override_dh_clistrip:
 
 override_dh_auto_build:
-	dotnet publish --configuration $(CONFIG) --output='$(CURDIR)/usr/lib/jellyfin/bin' --self-contained --runtime linux-x64
+	dotnet publish MediaBrowser.Server.Mono/EmbyServer.csproj --configuration $(CONFIG) --output='$(CURDIR)/usr/lib/jellyfin/bin' --self-contained --runtime linux-x64
```

Comment:
Only publish the EmbyServer project instead of the whole solution. With the `--self-contained` flag set, test projects will fail to publish, and we only need to publish this 1 project anyway.